### ISSUE TITLE
Liquibase migration fails for v0.27.0.RC1, v0.26.2 works

### ISSUE THREAD

**Issue by sunshineo** [metabase/metabase-buildpack#3]
```
2017-11-24T09:00:54.527253+00:00 heroku[web.1]: Starting process with command `./bin/start`
2017-11-24T09:00:57.116962+00:00 app[web.1]: Setting JAVA_TOOL_OPTIONS defaults based on dyno size. Custom settings will override them.
2017-11-24T09:00:57.196878+00:00 app[web.1]: Picked up JAVA_TOOL_OPTIONS: -Xmx300m -Xss512k -Dfile.encoding=UTF-8 
2017-11-24T09:01:13.983338+00:00 app[web.1]: 11-24 09:01:13 [1mINFO metabase.util[0m :: Loading Metabase...
2017-11-24T09:01:35.492942+00:00 app[web.1]: 11-24 09:01:35 [1mINFO util.encryption[0m :: DB details encryption is DISABLED for this Metabase instance. 🔓 
2017-11-24T09:01:35.492954+00:00 app[web.1]: See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
2017-11-24T09:02:00.113634+00:00 app[web.1]: 11-24 09:02:00 [1mINFO metabase.core[0m :: Starting Metabase in STANDALONE mode
2017-11-24T09:02:00.151902+00:00 app[web.1]: 11-24 09:02:00 [1mINFO metabase.core[0m :: Launching Embedded Jetty Webserver with config:
2017-11-24T09:02:00.151905+00:00 app[web.1]:  {:port 38782}
2017-11-24T09:02:00.151906+00:00 app[web.1]: 
2017-11-24T09:02:00.282933+00:00 app[web.1]: 11-24 09:02:00 [1mINFO metabase.core[0m :: Starting Metabase version v0.27.0.RC1 (2c0ccb6 release-0.27.0) ...
2017-11-24T09:02:00.284318+00:00 app[web.1]: 11-24 09:02:00 [1mINFO metabase.core[0m :: System timezone is 'Etc/UTC' ...
2017-11-24T09:02:00.787578+00:00 heroku[web.1]: State changed from starting to up
2017-11-24T09:02:08.639644+00:00 heroku[web.1]: Process running mem=518M(101.3%)
2017-11-24T09:02:08.639644+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:02:15.502112+00:00 app[web.1]: 11-24 09:02:15 [1mWARN metabase.driver[0m :: No -init-driver function found for 'metabase.driver.google'
2017-11-24T09:02:18.458228+00:00 app[web.1]: 11-24 09:02:18 [1mINFO metabase.core[0m :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
2017-11-24T09:02:18.465764+00:00 app[web.1]: 11-24 09:02:18 [1mINFO metabase.db[0m :: [36mVerifying postgres Database Connection ...[0m
2017-11-24T09:02:19.202782+00:00 app[web.1]: 11-24 09:02:19 [1mINFO metabase.db[0m :: Verify Database Connection ...  ✅
2017-11-24T09:02:19.203341+00:00 app[web.1]: 11-24 09:02:19 [1mINFO metabase.db[0m :: Running Database Migrations...
2017-11-24T09:02:19.307381+00:00 app[web.1]: 11-24 09:02:19 [1mINFO metabase.db[0m :: Setting up Liquibase...
2017-11-24T09:02:19.826138+00:00 app[web.1]: 11-24 09:02:19 [1mINFO metabase.db[0m :: Liquibase is ready.
2017-11-24T09:02:19.826609+00:00 app[web.1]: 11-24 09:02:19 [1mINFO metabase.db[0m :: Checking if Database has unrun migrations...
2017-11-24T09:02:29.480405+00:00 heroku[web.1]: Process running mem=533M(104.2%)
2017-11-24T09:02:29.480534+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:02:50.570148+00:00 heroku[web.1]: Process running mem=535M(104.5%)
2017-11-24T09:02:50.570148+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:03:11.206629+00:00 heroku[web.1]: Process running mem=536M(104.8%)
2017-11-24T09:03:11.206767+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:03:32.924175+00:00 heroku[web.1]: Process running mem=537M(104.9%)
2017-11-24T09:03:32.924246+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:03:53.693147+00:00 heroku[web.1]: Process running mem=538M(105.2%)
2017-11-24T09:03:53.693252+00:00 heroku[web.1]: Error R14 (Memory quota exceeded)
2017-11-24T09:04:11.130652+00:00 app[web.1]: liquibase.exception.ChangeLogParseException: Error parsing liquibase.yaml
2017-11-24T09:04:11.130962+00:00 app[web.1]: 	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:98)
2017-11-24T09:04:11.130993+00:00 app[web.1]: 	at liquibase.Liquibase.getDatabaseChangeLog(Liquibase.java:229)
2017-11-24T09:04:11.131058+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1183)
2017-11-24T09:04:11.131117+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1176)
2017-11-24T09:04:11.131191+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1172)
2017-11-24T09:04:11.131243+00:00 app[web.1]: 	at metabase.db$has_unrun_migrations_QMARK_.invokeStatic(db.clj:135)
2017-11-24T09:04:11.131303+00:00 app[web.1]: 	at metabase.db$has_unrun_migrations_QMARK_.invoke(db.clj:129)
2017-11-24T09:04:11.131379+00:00 app[web.1]: 	at metabase.db$migrate_up_if_needed_BANG_.invokeStatic(db.clj:159)
2017-11-24T09:04:11.131436+00:00 app[web.1]: 	at metabase.db$migrate_up_if_needed_BANG_.invoke(db.clj:151)
2017-11-24T09:04:11.131534+00:00 app[web.1]: 	at metabase.db$migrate_BANG_$fn__21163.invoke(db.clj:247)
2017-11-24T09:04:11.131535+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:714)
2017-11-24T09:04:11.131584+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
2017-11-24T09:04:11.131639+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:740)
2017-11-24T09:04:11.131707+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
2017-11-24T09:04:11.131741+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:697)
2017-11-24T09:04:11.131798+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
2017-11-24T09:04:11.131854+00:00 app[web.1]: 	at metabase.db$migrate_BANG_.invokeStatic(db.clj:235)
2017-11-24T09:04:11.131911+00:00 app[web.1]: 	at metabase.db$migrate_BANG_.invoke(db.clj:218)
2017-11-24T09:04:11.131956+00:00 app[web.1]: 	at metabase.db$run_schema_migrations_BANG_.invokeStatic(db.clj:365)
2017-11-24T09:04:11.132011+00:00 app[web.1]: 	at metabase.db$run_schema_migrations_BANG_.invoke(db.clj:360)
2017-11-24T09:04:11.144228+00:00 app[web.1]: 	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:383)
2017-11-24T09:04:11.144274+00:00 app[web.1]: 	at metabase.db$setup_db_BANG_.doInvoke(db.clj:376)
2017-11-24T09:04:11.144331+00:00 app[web.1]: 	at clojure.lang.RestFn.invoke(RestFn.java:421)
2017-11-24T09:04:11.144380+00:00 app[web.1]: 	at metabase.core$init_BANG_.invokeStatic(core.clj:108)
2017-11-24T09:04:11.144440+00:00 app[web.1]: 	at metabase.core$init_BANG_.invoke(core.clj:87)
2017-11-24T09:04:11.144504+00:00 app[web.1]: 	at metabase.core$start_normally.invokeStatic(core.clj:194)
2017-11-24T09:04:11.144565+00:00 app[web.1]: 	at metabase.core$start_normally.invoke(core.clj:188)
2017-11-24T09:04:11.144599+00:00 app[web.1]: 	at metabase.core$_main.invokeStatic(core.clj:292)
2017-11-24T09:04:11.144655+00:00 app[web.1]: 	at metabase.core$_main.doInvoke(core.clj:287)
2017-11-24T09:04:11.144711+00:00 app[web.1]: 	at clojure.lang.RestFn.invoke(RestFn.java:397)
2017-11-24T09:04:11.144773+00:00 app[web.1]: 	at clojure.lang.AFn.applyToHelper(AFn.java:152)
2017-11-24T09:04:11.144827+00:00 app[web.1]: 	at clojure.lang.RestFn.applyTo(RestFn.java:132)
2017-11-24T09:04:11.144895+00:00 app[web.1]: 	at metabase.core.main(Unknown Source)
2017-11-24T09:04:11.145219+00:00 app[web.1]: Caused by: liquibase.exception.SetupException: Error parsing migrations/000_migrations.yaml
2017-11-24T09:04:11.145278+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.includeAll(DatabaseChangeLog.java:448)
2017-11-24T09:04:11.145327+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.handleChildNode(DatabaseChangeLog.java:352)
2017-11-24T09:04:11.145392+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.load(DatabaseChangeLog.java:282)
2017-11-24T09:04:11.145464+00:00 app[web.1]: 	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:91)
2017-11-24T09:04:11.145521+00:00 app[web.1]: 	... 32 more
2017-11-24T09:04:11.145695+00:00 app[web.1]: Caused by: liquibase.exception.ChangeLogParseException: Error parsing migrations/000_migrations.yaml
2017-11-24T09:04:11.145752+00:00 app[web.1]: 	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:98)
2017-11-24T09:04:11.145808+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.include(DatabaseChangeLog.java:478)
2017-11-24T09:04:11.145866+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.includeAll(DatabaseChangeLog.java:445)
2017-11-24T09:04:11.145930+00:00 app[web.1]: 	... 35 more
2017-11-24T09:04:11.146086+00:00 app[web.1]: Caused by: java.lang.OutOfMemoryError: GC overhead limit exceeded
2017-11-24T09:04:11.146141+00:00 app[web.1]: 	at java.util.Hashtable.addEntry(Hashtable.java:435)
2017-11-24T09:04:11.146174+00:00 app[web.1]: 	at java.util.Hashtable.put(Hashtable.java:476)
2017-11-24T09:04:11.146212+00:00 app[web.1]: 	at sun.security.util.SignatureFileVerifier.updateSigners(SignatureFileVerifier.java:831)
2017-11-24T09:04:11.146261+00:00 app[web.1]: 	at sun.security.util.SignatureFileVerifier.processImpl(SignatureFileVerifier.java:349)
2017-11-24T09:04:11.146329+00:00 app[web.1]: 	at sun.security.util.SignatureFileVerifier.process(SignatureFileVerifier.java:263)
2017-11-24T09:04:11.146364+00:00 app[web.1]: 	at java.util.jar.JarVerifier.processEntry(JarVerifier.java:318)
2017-11-24T09:04:11.146407+00:00 app[web.1]: 	at java.util.jar.JarVerifier.update(JarVerifier.java:230)
2017-11-24T09:04:11.146454+00:00 app[web.1]: 	at java.util.jar.JarFile.initializeVerifier(JarFile.java:383)
2017-11-24T09:04:11.146518+00:00 app[web.1]: 	at java.util.jar.JarFile.getInputStream(JarFile.java:450)
2017-11-24T09:04:11.146553+00:00 app[web.1]: 	at sun.net.www.protocol.jar.JarURLConnection.getInputStream(JarURLConnection.java:162)
2017-11-24T09:04:11.146645+00:00 app[web.1]: 	at liquibase.resource.ClassLoaderResourceAccessor.getResourcesAsStream(ClassLoaderResourceAccessor.java:50)
2017-11-24T09:04:11.146725+00:00 app[web.1]: 	at liquibase.util.StreamUtil.singleInputStream(StreamUtil.java:186)
2017-11-24T09:04:11.146726+00:00 app[web.1]: 	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:26)
2017-11-24T09:04:11.146761+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.include(DatabaseChangeLog.java:478)
2017-11-24T09:04:11.146812+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.includeAll(DatabaseChangeLog.java:445)
2017-11-24T09:04:11.146859+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.handleChildNode(DatabaseChangeLog.java:352)
2017-11-24T09:04:11.146930+00:00 app[web.1]: 	at liquibase.changelog.DatabaseChangeLog.load(DatabaseChangeLog.java:282)
2017-11-24T09:04:11.146964+00:00 app[web.1]: 	at liquibase.parser.core.yaml.YamlChangeLogParser.parse(YamlChangeLogParser.java:91)
2017-11-24T09:04:11.147011+00:00 app[web.1]: 	at liquibase.Liquibase.getDatabaseChangeLog(Liquibase.java:229)
2017-11-24T09:04:11.147059+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1183)
2017-11-24T09:04:11.147141+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1176)
2017-11-24T09:04:11.147175+00:00 app[web.1]: 	at liquibase.Liquibase.listUnrunChangeSets(Liquibase.java:1172)
2017-11-24T09:04:11.147213+00:00 app[web.1]: 	at metabase.db$has_unrun_migrations_QMARK_.invokeStatic(db.clj:135)
2017-11-24T09:04:11.147261+00:00 app[web.1]: 	at metabase.db$has_unrun_migrations_QMARK_.invoke(db.clj:129)
2017-11-24T09:04:11.147337+00:00 app[web.1]: 	at metabase.db$migrate_up_if_needed_BANG_.invokeStatic(db.clj:159)
2017-11-24T09:04:11.147368+00:00 app[web.1]: 	at metabase.db$migrate_up_if_needed_BANG_.invoke(db.clj:151)
2017-11-24T09:04:11.147404+00:00 app[web.1]: 	at metabase.db$migrate_BANG_$fn__21163.invoke(db.clj:247)
2017-11-24T09:04:11.147452+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:714)
2017-11-24T09:04:11.147542+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
2017-11-24T09:04:11.147543+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:740)
2017-11-24T09:04:11.147574+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
2017-11-24T09:04:11.147613+00:00 app[web.1]: 	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:697)
2017-11-24T09:04:11.153771+00:00 app[web.1]: 11-24 09:04:11 [1mERROR metabase.core[0m :: Metabase Initialization FAILED:  Error parsing liquibase.yaml
2017-11-24T09:04:11.176494+00:00 app[web.1]: 11-24 09:04:11 [1mINFO metabase.core[0m :: Metabase Shutting Down ...
2017-11-24T09:04:11.177113+00:00 app[web.1]: 11-24 09:04:11 [1mINFO metabase.core[0m :: Metabase Shutdown COMPLETE
2017-11-24T09:04:11.795550+00:00 heroku[web.1]: State changed from up to crashed
2017-11-24T09:04:11.798274+00:00 heroku[web.1]: State changed from crashed to starting
2017-11-24T09:04:11.780865+00:00 heroku[web.1]: Process exited with status 1
```

**Comment by senior** [comment]
This looks to be a duplicate of https://github.com/metabase/metabase/issues/6463

**Comment by salsakran** [comment]
As @senior mentioned, this is a dupe of an issue in the main repository, and in general, bug reports or errors should be posted there.

We just released 0.27.2 which fixes this.
